primitives/rpc/evm-tracing-events/src/evm.rs

```diff
@@ -210,6 +249,16 @@ impl<'a> From<evm::tracing::Event<'a>> for EvmEvent {
 				is_static,
 				context: context.clone().into(),
 			},
+			// [TODO] Need to check...
```

Comment:
Please help to check. If that's not enabled now, please help to update related comments